virtual-hyperscript/hooks/attribute-hook.js

```diff
@@ -4,7 +4,7 @@ module.exports = AttributeHook;
 
 function AttributeHook(namespace, value) {
     if (!(this instanceof AttributeHook)) {
-        return new AttributeHook(value);
+        return new AttributeHook(namespace, value);
```

Comment:
semver question... is this a breaking change to a public API?  specifically, it's obviously a breaking change but is it a public API?